mozilla/kuma
# fix bug 873327 - use librabbitmq for celery

To test:
1. Run `manage.py celeryd`
2. Set `CELERY_ALWAYS_EAGER = False` in `settings_local.py`
3. Perform an operation that triggers a celery task and confirm it executes (e.g., edit a watched article and verify email output, or edit a page and verify elastic search index update)
